frontend/src/metabase/collections/components/CollectionEmptyState/CollectionEmptyState.tsx

```diff
@@ -12,22 +12,26 @@ import {
 } from "./CollectionEmptyState.styled";
 
 export interface CollectionEmptyStateProps {
-  collectionId?: CollectionId;
+  collection?: Collection;
 }
 
 const CollectionEmptyState = ({
-  collectionId,
+  collection,
 }: CollectionEmptyStateProps): JSX.Element => {
+  const canWrite = collection?.can_write ?? false;
```

Comment:
this is the most nitpicky code comment I have ever made

```suggestion
  const canWrite = !!collection?.can_write;
```